JourneysSquad/JourneysSquadTests/Model/BillValidationErrorTests.swift

```diff
@@ -0,0 +1,58 @@
+import XCTest
+
+final class BillValidationErrorTests: XCTestCase {
+    func testErrorDescriptionPresentsInAPIForTooLongError() {
+        _ = BillValidationError.tooLongParticipantName(name: "Abu Ibn .....").errorDescription
```

Comment:
nit:

Why do you need this strange names here?
```suggestion
        _ = BillValidationError.tooLongParticipantName(name: "").errorDescription
```

Reply:
Please review the rest of code.

Reply:
Fixed